test/common_utils.py

```diff
@@ -284,8 +289,29 @@ def __init__(
         mae=False,
         **other_parameters,
     ):
-        if all(isinstance(input, PIL.Image.Image) for input in [actual, expected]):
-            actual, expected = (to_image(input) for input in [actual, expected])
+        # Convert PIL images to tv_tensors.Image (regardless of what the other is)
+        if isinstance(actual, PIL.Image.Image):
+            actual = to_image(actual)
+        if isinstance(expected, PIL.Image.Image):
+            expected = to_image(expected)
+
+        # Convert CV-CUDA tensors to torch.Tensor (regardless of what the other is)
+        try:
+            import cvcuda
+            from torchvision.transforms.v2.functional import cvcuda_to_tensor
```

Comment:
Let's use the `_is_cvcuda_available()` instead of a try/except. You can then call `cvcuda = _import_cvcuda()` which is safer and less surprising than a raw `import cvcuda`.